templates/discussions-dashboard.php

```diff
@@ -0,0 +1,185 @@
+		$path_parts     = explode( '/', $parts['path'] );
+
+		$linking_comment_original_id = array_pop( $path_parts );
+
+		if ( ! isset( $bulk_comments[ $linking_comment_original_id ] ) ) {
+			$bulk_comments[ $linking_comment_original_id ] = array();
+		}
+
+		$bulk_comments[ $linking_comment_original_id ][] = $_comment;
+		continue;
+	}
+
+	if ( ! isset( $comments_by_post_id[ $_comment->comment_post_ID ] ) ) {
+		$comments_by_post_id[ $_comment->comment_post_ID ] = array();
+	}
+
+
+	$comments_by_post_id[ $_comment->comment_post_ID ][] = $_comment;
+
+	if ( ! isset( $latest_comment_date_by_post_id[ $_comment->comment_post_ID ] ) ) {
+		$latest_comment_date_by_post_id[ $_comment->comment_post_ID ] = $_comment->comment_date;
+	} elseif ( $latest_comment_date_by_post_id[ $_comment->comment_post_ID ] < $_comment->comment_date ) {
+		$latest_comment_date_by_post_id[ $_comment->comment_post_ID ] = $_comment->comment_date;
+	}
+}
+
+// If the referenced comment is not in the current batch of comments we need to re-add it.
+foreach ( $bulk_comments as $original_id => $_post_id ) {
+	if ( ! isset( $comments_by_post_id[ $_comment->comment_post_ID ] ) ) {
+		$linked_comment = $_comment->comment_content;
+		$parts          = wp_parse_url( $linked_comment );
+		$comment_id     = intval( str_replace( 'comment-', '', $parts['fragment'] ) );
+		if ( $comment_id ) {
+			$comments_by_post_id[ $_comment->comment_post_ID ][] = get_comment( $comment_id );
+		}
+	}
+}
+
+uasort(
+	$comments_by_post_id,
+	function( $a, $b ) use ( $latest_comment_date_by_post_id ) {
+		return $latest_comment_date_by_post_id[ $b->comment_post_ID ] <=> $latest_comment_date_by_post_id[ $a->comment_post_ID ];
+	}
+);
+
+$args = array(
+	'style'            => 'ul',
+	'type'             => 'comment',
+	'callback'         => 'gth_discussion_callback',
+	'reverse_children' => false,
+);
+
+
+
+?>
+
+<table id="translations" class="translations clear">
+	<thead class="discussions-table-head">
+	<tr>
+		<th>Original string</th>
+		<th>Comment</th>
+		<th>Author</th>
+		<th>Submitted on</th>
+	</tr>
+	</thead>
+	<tbody>
+		<?php
+		foreach ( $comments_by_post_id as $_post_id => $post_comments ) {
+			$original_id = Helper_Translation_Discussion::get_original_from_post_id( $_post_id );
+			if ( ! $original_id ) {
+				continue;
+			}
+
+			$original             = GP::$original->get( $original_id );
+			$first_comment        = reset( $post_comments );
+			$no_of_other_comments = count( $post_comments ) - 1;
+			?>
+			<tr>
+				<td>
+					<?php
+					echo esc_html( $original->singular );
+					if ( isset( $bulk_comments[ $original_id ] ) ) {
+						?>
+						<details>
+							<summary class="other-comments">
+							<?php
+							/* translators: number of other originals in the bulk rejection. */
+							printf( '+ ' . _n( '%s Other', '%s Others', count( $bulk_comments[ $original_id ] ) ), number_format_i18n( count( $bulk_comments[ $original_id ] ) ) );
```

Comment:
Could we call this "more bulk rejections" see this screenshot, I think it makes better use of the space:

![Screenshot 2022-09-02 at 14 53 34](https://user-images.githubusercontent.com/203408/188150230-51342220-778e-4f52-8330-42eb42b2afe7.png)

Reply:
How about something like `+ 3 more strings in this bulk rejection` or  `+ 3 more strings in bulk rejection`?

Reply:
`+ 2 Comments` should better be: `Thread with 2 comments`